public class Input {

    public void testOne() throws Exception{
        int i = 0;
        int c=1/i;
        System.out.println(c+"第2次测试");
    }
}
